database/factories/EmployeeFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Lightit\Backoffice\Employee\Domain\Models\Employee;
+
+/**
+ * @extends Factory<\Lightit\Backoffice\Employee\Domain\Models\Employee>
+ */
+class EmployeeFactory extends Factory
+{
+
+    /**
+     * @var class-string<\Lightit\Backoffice\Employee\Domain\Models\Employee>
+     */
+    protected $model = Employee::class;
+    public function definition(): array
+    {
+        return [
+            'name' => $this->faker->company(),
+            'email' => $this->faker->unique()->companyEmail(),
```

Comment:
if im not mistaken, you can use `faker()->` instead of `$this->faker->` in the latest laravel version